### DISCUSSION TITLE
Question about West Virginia senior citizen deduction

### DISCUSSION THREAD

**Discussion by martinholmer** [PolicyEngine/policyengine-us#3786] [Q&A] [answered]
In the 0.639.0 version, the code for the `wv_senior_citizen_disability_deduction_person` variable is as follows:

https://github.com/PolicyEngine/policyengine-us/blob/babd6ea6efa0553ebc460bc9de48f2a2ebf1b299/policyengine_us/variables/gov/states/wv/tax/income/subtractions/senior_citizen_disability_deduction/wv_senior_citizen_disability_deduction_person.py#L4-L28

I don't understand the logic of line 20 or the following three-line comment.   The form caps at $8000 not AGI but AGI less other modifications.  Because I don't understand the "we can mathematically derive this condition" business, I wonder if the logic is incorrect.   Can you explain the comment to me?

**Comment by PavelMakarchuk** [comment, marked as answer]
@martinholmer the comment explains that we only implicitly model the following law passage:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/0d2f9bd1-5977-444e-869a-fc0e5005fbef)

If the amount of `wv_senior_citizen_disability_deduction_total_modifications` increase over $8,000 per person, that person should be ineligible. 
Instead of including another parameter for the $8,000 limit and checking the modifications against that, we reduce the amount of applicable capped AGI (max $8,000) by the amount of modifications until it reaches 0. 

Let me know if we are making a mistake here thinking about the cap.

**Comment by martinholmer** [comment]
@PavelMakarchuk,  Thanks for the explanation in issue #3773.
I am closing this issue now given that you have provided an explanation of the code.
